.pre-commit-config.yaml

```diff
@@ -31,16 +31,16 @@
                 \.eslintrc\.js
             )$
     pass_filenames: true
-  - id: isort
-    name: isort-local
-    entry : isort
-    language: python
-    types: [python]
-    exclude: .+/(settings|migrations)/.+
-    pass_filenames: true
   - id: missing-migrations
     name: missing-migrations-local
     entry: python manage.py has_missing_migrations
     language: system
     always_run: true
     pass_filenames: false
+  - id: prospector
+    name: prospector-local
+    entry: prospector
```

Comment:
Add this to prospector entry: `prospector --messages-only --zero-exit`. It'll return only linter messages and 0 if everything went okay. It provides a clearer output.

Reply:
https://prospector.landscape.io/en/master/usage.html#Named%20Arguments

Reply:
Can you fix this and merge @rsarai ?